### PR TITLE
CORE-8414: HTTP RPC libraries update

### PR DESCRIPTION
**Pull request by vkolomeyko**
Javalin to 4.6.7
Swagger UI to 4.15.5
Nimbus to 9.43.1
Unirest to 3.14.0
Jetty to 9.4.49.v20220914

Tested locally with AzureAD OAuth2 SSO.

### CODE REVIEW

**Review by ronanbrowne — APPROVED** [review]
LGTM

**Review by pmck91 — APPROVED** [review]
LGTM

**Review by benmcmahon100 — APPROVED** [review]

File: gradle.properties [hunk 1]
```diff
@@ -103,13 +103,13 @@ postgresDriverVersion=42.4.1
 slingVersion=3.3.2
 
 # HTTP RPC dependency versions
-javalinVersion = 4.6.4
+javalinVersion = 4.6.7
```
Comment (sprintengine) [inline gradle.properties:106]: I think you mentioned last week, but I notice Javalin is up to 5.2.0 now, is upgrading to 5 on our radar?

Comment (vkolomeyko) [reply gradle.properties:106]: Yeap, there is a Jira for that: https://r3-cev.atlassian.net/browse/CORE-8415
(but it is more involved)

**Review by conal-smith — APPROVED** [review]
LGTM

**Pull request merged**